File: app/src/main/java/com/sixbeeps/uniquity/data/UnicodeCharacterAlias.java
package com.sixbeeps.uniquity.data;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class UnicodeCharacterAlias {
    @PrimaryKey(autoGenerate = true)
    public int id;

    @NonNull
    public String codepoint = "";

    @NonNull
    public String alias = "";
}

File: app/src/main/java/com/sixbeeps/uniquity/data/UnicodeCharacterAlias.kt
package com.sixbeeps.uniquity.data

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
class UnicodeCharacterAlias {
    @JvmField
    @PrimaryKey(autoGenerate = true)
    var id: Int = 0

    @JvmField
    var codepoint: String = ""

    @JvmField
    var alias: String = ""
}

